id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/builder/RestRequestBuilder.java

```diff
@@ -108,7 +109,7 @@ public RestRequestDTO buildRequest(RestServicesConstants restService, Object req
 		checkHttpMethod(request, httpMethod);
 
 		if (requestBody != null) {
-			if (!headers.getContentType().includes(MediaType.MULTIPART_FORM_DATA)) {
+			if (!Objects.requireNonNull(headers.getContentType()).includes(MediaType.MULTIPART_FORM_DATA)) {
```

Comment:
instead of using Objects.requireNonNull, add condition to check for Objects.nonNull